### PR TITLE
feat(Homepage): Main page / Home page UI completion

### PR DESCRIPTION
**Pull request by CologneX**
merge dong

### CODE REVIEW

**Comment by CologneX** [comment]
LGTM!

File: MoneyCompass/Onboarding/FillProfileView.swift [hunk 1]
```diff
@@ -85,6 +85,7 @@ struct FillProfileView: View {
             let defaults = UserDefaults.standard
             defaults.set(firstName, forKey: "firstName")
             defaults.set(lastName, forKey: "lastName")
+            defaults.set(balance, forKey: "balance")
```
Comment (Tus1688) [inline MoneyCompass/Onboarding/FillProfileView.swift]: Balance shouldnt be saved into user defaults

File: MoneyCompass/Transaction/SavingGoalsView.swift [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        self.name
+    }
+    var name: String
+    var price: Double
+    var progress: Double
+}
+
+struct SavingGoalsView: View {
+    @State var progress: Double = 1
+    @State var goals: [Goals] = [
```
Comment (Tus1688) [inline MoneyCompass/Transaction/SavingGoalsView.swift]: Try to fetch saving goals from BudgetTarget  (core data) instead of showing dummy data

**Review by Tus1688 — CHANGES_REQUESTED** [review]

**Comment by CologneX** [comment]
Cancel merge, commit updates

**Review by Tus1688 — APPROVED** [review]

**Pull request merged**